var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -55,6 +55,13 @@ class Rust(Package):
         default=True,
         description='Install Rust source files'
     )
+    variant(
+        'extra_targets',
+        default=(),
+        values=('nvptx64-nvidia-cuda',),
```

Comment:
So I tested, and not specifying `values` seems to work to allow specifying anything, so I think we should maybe just not proscribe allowed values? 

```suggestion
```